_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -59,36 +59,20 @@ By setting the `tabindex` [attribute value][] of an `iframe` element to `-1` or
 
 #### Passed Example 1
 
-This `iframe` element contains no content that is part of [sequential focus navigation][].
-
-```html
-<iframe tabindex="-1" srcdoc="<h1>Hello world</h1>"></iframe>
-```
-
-#### Passed Example 2
-
 This `iframe` element contains a link that is not part of [sequential focus navigation][] because of its `tabindex`.
```

Comment:
Should we mark also this example as inapplicable now, due to the new definition "This rule applies to any `iframe` element that [contains](#akn7bn:contain) elements that are both [visible][] and part of the [sequential focus navigation][]."?

We should add instead an example with an iframe with no tabindex attribute (or tabindex="0", or both) and both visible and SFN  content.